.gitignore

```diff
@@ -47,4 +47,4 @@ kubeconfig
 
 # Ignore output manifests
 cmd/clusterctl/examples/aws/out
-cmd/clusterctl/examples/aws/provider-components-base-dev.yaml
+cmd/clusterctl/examples/aws/provider-components-base.yaml
```

Comment:
both -dev and this one are generated files. Neither of which should have been checked in